timemachine/utils.py

```diff
@@ -0,0 +1,28 @@
+import hashlib
+
+from rdkit import Chem
+
+from timemachine.fe import functional
+
+
+def bind_potentials(potentials, params):
+    """modifies potentials in-place"""
+    for U, p in zip(potentials, params):
+        U.bind(p)
+
+
+def construct_potential(potentials, params):
```

Comment:
nit: type annotation (and maybe a more descriptive name) seems particularly helpful here

Reply:
On second thought, seems better to just inline this in https://github.com/proteneer/timemachine/pull/718/commits/07e6c98310e6bb413670543b3b2063a633d8c543 , since only used once